.production.env

```diff
@@ -0,0 +1 @@
+PORT=7000
```

Comment:
Please add new line in the end of the file.

Reply:
All files were fixed